src/main/java/app/chatbot/Ui.java

```diff
@@ -0,0 +1,53 @@
+package app.chatbot;
+
+import java.util.Scanner;
+
+public class Ui {
+    private static final String LOGO = "       _             _    _    _                _                \n"
+            + "  ___ | |__    __ _ | |_ | |_ (_) _ __    __ _ | |_  ___   _ __  \n"
+            + " / __|| '_ \\  / _` || __|| __|| || '_ \\  / _` || __|/ _ \\ | '_ \\ \n"
+            + "| (__ | | | || (_| || |_ | |_ | || | | || (_| || |_| (_) || | | |\n"
+            + " \\___||_| |_| \\__,_| \\__| \\__||_||_| |_| \\__, | \\__|\\___/ |_| |_|\n"
+            + "                                         |___/                   ";
+
+    private Scanner sc;
+
+    Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    public void showWelcome() {
```

Comment:
Not sure if this is the latest commit but some public methods are missing JavaDocs while some are present.